extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -584,12 +622,12 @@ public function replyToMessage() {
 		$this->response->setVal( 'status', true );
 
 		$parentId = $this->request->getVal( 'parent' );
-		$parentTitle = Title::newFromId( $parentId );
+		$parentTitle = Title::newFromID( $parentId );
 		$debugParentDB = 'from slave';   // tracing bug 95249
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)